### ISSUE TITLE
Windows heating

### ISSUE THREAD

**Issue by Libar19** [adhyh/homebridge-vwconnectid#20]
Hello. Congratulations for your plugin.
But I have problem. It's possible to enable only Windows heating?  Because actualy only climatisation work. Before With weconnet, I can activate Clim or Windows heating or both. 
Thank's very much for your work.

**Comment by adhyh** [comment]
Hi,

Thanks for the suggestion. Unfortunately with the current API it is not possible. This option is also not available in the Volkswagen app, so probably this functionality was removed from the API at some point.

**Comment by Libar19** [comment]
If this feature is available on the app ![image](https://github.com/user-attachments/assets/3146bc1b-4f66-464a-9747-cd622dbf0a10)

**Comment by adhyh** [comment]
Looks like your car is fancier than mine :). Unfortunately in my ID.4 it is not available, so I can't inspect the backend communication with the app to see what happens exactly.

![app](https://github.com/user-attachments/assets/f2807911-94b3-4c53-9cca-0a9bc7896cec)

**Comment by Libar19** [comment]
what are you inspecting with? i try "charles proxy" but it don't work. I want to try to see what is the command that is sent from my iphone and try to integrate it

**Comment by adhyh** [comment]
I use Wireshark.

**Comment by adhyh** [comment]
Unfortunately I don't have the time to walk you through the process, but look for mitmproxy tutorial. Mitmproxy captures the iphone traffic, decrypts the ssl and shows on screen.

**Comment by Libar19** [comment]
no problem. I use a version of your plugin that I modified because many things did not work for my vehicle.
for the api I also modified them because the refresh times made the volkswagen application crash. so you do not need to put it in your version I will do it myself 🙂

**Comment by Libar19** [comment]
so I am analyzing HTTPS requests, but I have a problem with mitmproxy

i have installed the certificate and I am confident in my iphone, I have set the proxy of the mac that uses mitmproxy and I get the traffic visualization but is it normal that the server must be unreachable? as soon as I deactivate the proxy settings in my iphone the communication returns in the volkswagen application ?

**Comment by adhyh** [comment]
No, the communication should be normal. Can you take a look at the mitmproxy log (maybe increase verbosity level)?

**Comment by Libar19** [comment]
it is not verbosity level , i have this fault : warn: [21:40:58.888][192.168.1.35:56107] Client TLS handshake failed. Client and mitmproxy cannot agree on a TLS version to
use. You may need to adjust mitmproxy's tls_version_client_min option.

**Comment by adhyh** [comment]
I have not encountered this error, so I don't know. I remember I had to disable strict certificate checking or something like that, but that is probably unrelated to the error you are seeing.

**Comment by Macleykun** [comment]
I stumbled across this while looking into moving to homebridge from ha.
Would this repo help? I also have the newer module and it works https://github.com/robinostlund/homeassistant-volkswagencarnet/tree/master
